stdlib/experimental/window.go

```diff
@@ -0,0 +1,14 @@
+package experimental
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/stdlib/universe"
+)
+
+const WindowKind = "window"
```

Comment:
this constant isn't really needed now, maybe just use the string `"window"` directly below?